lldb/tools/lldb-dap/README.md

```diff
@@ -174,6 +174,20 @@ The default hostname being used `localhost`.
 }
 ```
 
+### Launching via `vscode://` URIs
+
+Debugging sessions can also be starting using special URIs.
+
+The `vscode://llvm-vs-code-extensions.lldb-dap/launch/config?config={launch-config}`
+URI accepts a [URL-encoded](https://en.wikipedia.org/wiki/Percent-encoding)
+JSON launch config.
+
+This is useful for integration with custom scripts to start debugging
+sessions. The URI might be printed to the terminal, potentially using
+[OSC-8 hyperlinks](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda),
+or passed to `vscode --open-url` or `xdg-open`, although mileage may vary
+depending on your specific debugging setup.
```

Comment:
I think your example should be with `code` and not `vscode`.
Besides that, mention that this doesn't work when using SSH remoting.
And also mention that workspace-based debug config variable resolution is not supported, as the arguments should be explicit and not use wildcards like `${workspaceFolder}`